### ISSUE TITLE
localize_entry fails

### ISSUE THREAD

**Issue by fkamras** [contentful/contentful_middleman#124]
I noticed some new fields in the synced data that contains the content type id and created and updated dates.
It seems like the localize_entry function is failing becuase of this, as it's trying to get the en-US field that does not exist there.
Seems like this is the problem as it's working once I remove the meta key before the localize_entry function but it would be better if I don't have to go through my whole code to add this.

I'm using the dl/upgrade-to-v4 branch

**Comment by fkamras** [comment]
I'm not sure if it's the same issue, but since yesterday all the data that I accessed with camel case is now only available with underscores....

**Comment by dlitvakb** [comment]
Hey @fkamras,

Regarding the first issue, I'll look into it ASAP, it looks like a bug.

Regarding the second issue, it's a documented change with version `2.0.0` -> https://github.com/contentful/contentful_middleman/blob/master/CHANGELOG.md#changed

Cheers

**Comment by brtdwchtr** [comment]
Bumping this... Any updates, @dlitvakb ?

If it might help, the problem code can be found on line 32 of helpers.rb

`return value.fetch(locale) if value.key? locale`
`return value.fetch(fallback_locale) `

It will throw an error when trying to localize for example the _meta property of a datafile
```
 :_meta:
    :content_type_id: sometype
    :updated_at: '2017-07-10T21:14:03+00:00'
    :created_at: '2017-07-06T11:24:38+00:00'
    :id: xYz123XyZ321zYX123
```

I was thinking the solution might be to rewrite localize_value like this:
```
def localize_value(value, locale, fallback_locale='en-US')
    if value.respond_to? :fetch
       return value.fetch(locale) if value.key? locale
       return value.fetch(fallback_locale) if value.key? fallback_locale
    end
    value
end
```
Unless just returning the original hash, if no language or fallback has been found, could present another problem?

**Comment by dlitvakb** [comment]
Fixed by #128 

Thanks!
